includes/Post_Type_Base.php

```diff
@@ -286,20 +286,52 @@ public function get_parent_controller(): WP_REST_Controller {
 		return $parent_controller;
 	}
 
+	/**
+	 * Retrieves the permalink for a post type archive.
+	 *
+	 * @since 1.14.0
+	 *
+	 * @return string|false The post type archive permalink. False if the post type
+	 *                      does not exist or does not have an archive.
+	 */
+	public function get_archive_link() {
+		$post_type_obj = $this->get_object();
+		if ( ! $post_type_obj instanceof WP_Post_Type ) {
+			return false;
+		}
+
+		return $this->get_archive_link_data( $post_type_obj->has_archive );
+	}
+
+	/**
+	 * Calls get_archive_link_data method to get the value of archive by forcing has_archive to true.
+	 *
+	 * @since 1.17.0
```

Comment:
```suggestion
	 * @since 1.18.0
```

Please check out the generally improved PHPDoc I added in #10442

The return description is not accurate and the short description above is not useful either.